flips/20210914-voting-mechanics-decentralized-ecosystems.md

```diff
@@ -0,0 +1,119 @@
+# Voting Mechanics in Decentralized Ecosystems
+
+| Status        | Proposed                                             |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [631](https://github.com/onflow/flow/pull/631)       |
+| **Author(s)** | Jeffrey Doyle (jeffrey.doyle@dapperlabs.com),        |
+| **Sponsor**   | Jeffrey Doyle (jeffrey.doyle@dapperlabs.com)         |
+| **Updated**   | 2021-09-14                                           |
+
+## Overview
+
+In decentralized organizations, governance is often constructed such that
+authority in the organization is distributed horizontally to many or all
+organizational participants, rather than vertically in such a way that grants
+some participants authority in decision making while others do not have a voice.
+
+The topic of decentralized governance has been explored many times across
+various blockchain projects. Often a DAO (Decentralized Autonomous Organization)
+is constructed, represented as a Smart Contract, which facilitates a structure
+of rules that outline how the organization performs actions upon a system it
+controls. When constructed, the DAO is given control of some system, and the
+ability to impact change upon that system. The actions performed by the DAO upon
+the system it controls are often _proposed_, and then _voted_ upon by members or
+participants of the organization. If conditions are met such that a vote is
+passed, the proposed action is carried out autonomously by the DAO.
+
+What a DAO can do is virtually limitless. However, the mechanic that enables
+community participation in the actions performed by a DAO, or any other
+decentralized ecosystem is contentious. There has yet to be discovered a
+universally accepted set of best practices for enabling decentralized, community
+driven governance in a decentralized system.
```

Comment:
```suggestion
decentralized ecosystem is contentious. A universally accepted set of best practices
for enabling decentralized, community driven governance in a decentralized system
is yet to be discovered.
```

Perhaps it sounds better this way around? 
Maybe a different verb could be used instead of _discovered_? _crafted_?